### PR TITLE
Xslt guide

### PR DESCRIPTION
**Pull request by saiskee**
Docs-only change:
Creates a guide for using the XSLT transformation.

### CODE REVIEW

File: docs/examples/xslt-guide/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.13.4
+
+RUN apk upgrade --update-cache \
+    && apk add ca-certificates \
+    && rm -rf /var/cache/apk/*
+
```
Comment (sam-heilbron) [inline docs/examples/xslt-guide/Dockerfile:6]: Doesn't hurt to have, but are these necessary for this example?

**Review by sam-heilbron — COMMENTED** [review]
This is looking great! Just small comments

File: docs/content/guides/traffic_management/request_processing/transformations/xslt_transformation/_index.md [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+        options:
+          autoHostRewrite: true
+EOF
+{{< /highlight >}}
+
+### Querying the SOAP service
+
+SOAP services communicate to clients via XML. We can query this SOAP service with an XML curl:
+```shell
+curl $(glooctl proxy url) -H "SOAPAction:findCity" -H "content-type:application/xml" \
```
[the remark this answers is not in the document]
Comment (saiskee) [reply docs/content/guides/traffic_management/request_processing/transformations/xslt_transformation/_index.md:133]: Yes it is, if you look at the SOAP application code, it's a arbitrary custom action string that you can provide to the server.

File: docs/examples/xslt-guide/Dockerfile (hunk 1 above)
Comment (saiskee) [reply docs/examples/xslt-guide/Dockerfile:6]: Might as well have them haha, we include them in other docker example images.

**Review by sam-heilbron — APPROVED** [review]

**Pull request merged**